projects/inc/transmitter.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <array>
+#include "cobs.h"
+#include "ch.h"
+#include "simulation.pb.h"
+
+namespace message {
+class Transmitter {
+    public:
+        Transmitter();
+        void start(tprio_t priority = (NORMALPRIO + 1));
```

Comment:
Okay. As a note, the user will also need to set STM32_USB_OTG_THREAD_PRIO. Maybe it's possible to change with this function call.